taskiq_sqs/aws.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+
+import urllib3  # boto3 peer dep (v1)
+
+ECS_CONTAINER_METADATA_URI = "http://169.254.170.2"
```

Comment:
What's this?

Reply:
ECS metadata service.

Reply:
Is it a globally accessible IP address? Per region? AWS credentials required?

Reply:
It's a link-local address, so not publicly routable. Not per-region. No auth tokens required.